Update docs with the recent changes (#35)
rhd-gitops-example/docs: journey/day1/README.md

* [odo pipelines bootstrap](../../commands/bootstrap)

> * [odo pipelines init](../../commands/init)

Reviewer: ```suggestion
```

> These commands are similar.  They differ in what gets generated.  The bootstrap command generates a functional GitOps setup including your first application.  The init command only generates CI/CD pipelines and associated resources.  This document describes how to bootstrap GitOps to deliver your first application.

Reviewer: ```suggestion
The bootstrap command generates a functional GitOps setup including your first application.

This document describes how to bootstrap GitOps to deliver your first application.
```

## Prerequisites

> **NOTE**: `odo pipelines` commands are hidden in `Expermential Mode`.  To enable `Expermential Mode` available, please set the `EXPERIEMENTAL` environment variable in the running terminal.

Reviewer: ```suggestion
**NOTE**: `odo pipelines` commands are experimental, please see [the odo documentation](https://docs.openshift.com/container-platform/4.5/cli_reference/developer_cli_odo/odo-release-notes.html#odo-technology-preview_odo-release-notes) for how to enable experimental mode.
```

  --gitops-repo-url https://github.com/<username>/gitops.git \
  --image-repo quay.io/<username>/<image-repo> \
  --dockercfgjson ~/Downloads/<username>-auth.json \
  --prefix tst \
  --output <path to write GitOps resources>
```

> Executing the above command will generate the GitOps directory and the required resources. 

Reviewer 2: The attached binary needed this, if we are changing this, are awe attaching a new binary?

Reviewer 3: yes

Reviewer 2: .. but does the current linked binary work without the flags`--sealed-secret-ns kube-system` and `--sealed-secrets-svc sealed-secrets-controller`?
